### PR TITLE
Upgraded nuget-packages within minor versions

### PR DESCRIPTION
**Pull request by Sceptic666**

### CODE REVIEW

File: Source/Bifrost.Client.Specs/Bifrost.Client.Specs.csproj [hunk 1]
```diff
@@ -39,30 +39,32 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\Solutions\packages\Castle.Core.3.3.3\lib\net45\Castle.Core.dll</HintPath>
     </Reference>
-    <Reference Include="Machine.Specifications">
-      <HintPath>..\Solutions\packages\Machine.Specifications.0.9.1\lib\net45\Machine.Specifications.dll</HintPath>
+    <Reference Include="Machine.Specifications, Version=0.11.0.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (bnordli) [inline Source/Bifrost.Client.Specs/Bifrost.Client.Specs.csproj:42]: Look in .Source\Solutions.nuget\packlages.config. There you'll find a solution wide reference that also should be updated.

File: Source/Bifrost.Default/Bifrost.Default.csproj [hunk 2]
```diff
@@ -48,26 +48,26 @@
       <HintPath>..\Solutions\packages\FluentValidation-Signed.6.2.1.0\lib\Net45\FluentValidation.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.AspNet.SignalR.Core, Version=2.2.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\Solutions\packages\Microsoft.AspNet.SignalR.Core.2.2.0\lib\net45\Microsoft.AspNet.SignalR.Core.dll</HintPath>
+    <Reference Include="Microsoft.AspNet.SignalR.Core, Version=2.2.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (bnordli) [inline Source/Bifrost.Default/Bifrost.Default.csproj:51]: The old version number is still present in various Web.config and app.config files.

File: Source/Bifrost.NHibernate/Bifrost.NHibernate.csproj [hunk 3]
```diff
@@ -40,17 +40,17 @@
     <AssemblyOriginatorKeyFile>..\..\Keys\Bifrost.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="FluentNHibernate, Version=2.0.1.0, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\Solutions\packages\FluentNHibernate.2.0.1.0\lib\net40\FluentNHibernate.dll</HintPath>
+    <Reference Include="FluentNHibernate, Version=2.0.3.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (bnordli) [inline Source/Bifrost.NHibernate/Bifrost.NHibernate.csproj:43]: You should update the version number in Bifrost.NHibernate.nuspec as well.

File: Source/Bifrost.NHibernate/Bifrost.NHibernate.csproj [hunk 4]
```diff
@@ -40,17 +40,17 @@
...
+      <Private>True</Private>
     </Reference>
     <Reference Include="Iesi.Collections, Version=4.0.0.0, Culture=neutral, PublicKeyToken=aa95f207798dfdb4, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\Solutions\packages\Iesi.Collections.4.0.1.4000\lib\net40\Iesi.Collections.dll</HintPath>
     </Reference>
     <Reference Include="NHibernate, Version=4.0.0.4000, Culture=neutral, PublicKeyToken=aa95f207798dfdb4, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\Solutions\packages\NHibernate.4.0.3.4000\lib\net40\NHibernate.dll</HintPath>
+      <HintPath>..\Solutions\packages\NHibernate.4.0.4.4000\lib\net40\NHibernate.dll</HintPath>
```
Comment (bnordli) [inline Source/Bifrost.NHibernate/Bifrost.NHibernate.csproj:52]: You should update the version number in Bifrost.NHibernate.nuspec as well.

File: Source/Bifrost.QuickStart/Bifrost.QuickStart.csproj [hunk 5]
```diff
@@ -159,8 +159,8 @@
     <Content Include="Scripts\jquery-2.2.3.min.map" />
     <None Include="Web.config.transform" />
     <Content Include="Scripts\bootstrap.js" />
-    <EmbeddedResource Include="Scripts\jquery.signalR-2.2.0.js" />
-    <EmbeddedResource Include="Scripts\jquery.signalR-2.2.0.min.js" />
+    <Content Include="Scripts\jquery.signalR-2.2.1.js" />
```
Comment (bnordli) [inline Source/Bifrost.QuickStart/Bifrost.QuickStart.csproj]: Keep these as embedded resources.

File: Source/Bifrost.Web/Bifrost.Web.csproj [hunk 6]
```diff
@@ -17,6 +17,8 @@
     <RestorePackages>true</RestorePackages>
     <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
     <TargetFrameworkProfile />
+    <NuGetPackageImportStamp>
```
Comment (bnordli) [inline Source/Bifrost.Web/Bifrost.Web.csproj]: Delete this.

File: Source/Bifrost.Web/Bifrost.Web.csproj [hunk 7]
```diff
@@ -84,8 +89,8 @@
       <HintPath>..\Solutions\packages\Microsoft.AspNet.Web.Optimization.1.1.3\lib\net40\System.Web.Optimization.dll</HintPath>
     </Reference>
     <Reference Include="WebActivatorEx, Version=2.0.0.0, Culture=neutral, PublicKeyToken=7b26dc2a43f6a0d4, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\Solutions\packages\WebActivatorEx.2.0.6\lib\net40\WebActivatorEx.dll</HintPath>
+      <HintPath>..\Solutions\packages\WebActivatorEx.2.2.0\lib\net40\WebActivatorEx.dll</HintPath>
```
Comment (bnordli) [inline Source/Bifrost.Web/Bifrost.Web.csproj:90]: Update version number in Bifrost.Web.nuspec.

File: Source/Bifrost.Web/Bifrost.Web.csproj [hunk 8]
```diff
@@ -212,8 +214,18 @@
     <EmbeddedResource Include="Scripts\knockout-3.4.0.debug.js" />
     <EmbeddedResource Include="Scripts\knockout-3.4.0.js" />
   </ItemGroup>
-  <Import Project="..\Solutions\packages\LessMsbuildTasks.1.0.7\tools\LessCompiler.targets" Condition="Exists('..\Solutions\packages\LessMsbuildTasks.1.0.7\tools\LessCompiler.targets')" />
+  <ItemGroup>
+    <Content Include="Scripts\jquery.signalR-2.2.1.js" />
```
Comment (bnordli) [inline Source/Bifrost.Web/Bifrost.Web.csproj]: Keep these as an embedded resource. You must also update the file name in ApplicationHttpHandler.

File: Source/Bifrost.Web/packages.config [hunk 9]
```diff
@@ -3,17 +3,17 @@
   <package id="Antlr" version="3.5.0.2" targetFramework="net45" />
   <package id="jQuery" version="2.2.3" targetFramework="net451" />
   <package id="knockoutjs" version="3.4.0" targetFramework="net451" />
-  <package id="LessMsbuildTasks" version="1.0.7" targetFramework="net45" />
-  <package id="Microsoft.AspNet.SignalR.Core" version="2.2.0" targetFramework="net45" />
-  <package id="Microsoft.AspNet.SignalR.JS" version="2.2.0" targetFramework="net45" />
-  <package id="Microsoft.AspNet.SignalR.SystemWeb" version="2.2.0" targetFramework="net45" />
+  <package id="LessMsbuildTasks" version="1.0.10" targetFramework="net451" />
```
Comment (bnordli) [inline Source/Bifrost.Web/packages.config:6]: Update Bifrost.Web.nuspec with all the new packages versions from this file. (Or remove the lines from the nuspec file and verify that the dependencies in the NuGet package stays the same, it should be picked up automatically..)

**Review by bnordli — CHANGES_REQUESTED** [review]

File: Source/Bifrost.Client.Specs/Bifrost.Client.Specs.csproj (hunk 1 above)
Comment (Sceptic666) [reply Source/Bifrost.Client.Specs/Bifrost.Client.Specs.csproj:42]: Runner is updated but do not have same version number as the other packages for Machine.Specifications.

**Comment by bnordli** [comment]
You are still missing ApplicationHttpHandler.cs, it references the old signalR file.

**Review by bnordli — APPROVED** [review]

**Pull request merged**